### PR TITLE
Bucket quotes by ETA, then sort by rate

### PR DESCRIPTION
**Pull request by artursapek**
Addresses #2889

As per discussion with @Wesleyleung - group quotes first by ETA, then sort by conversion rate. We will first show all quotes that land in under a minute, sorted by rate. Then, all the others also sorted by rate.

### CODE REVIEW

File: wormhole-connect/src/hooks/useSortedRoutesWithQuotes.ts [hunk 1]
```diff
@@ -134,3 +136,24 @@ export const useSortedRoutesWithQuotes = (): HookReturn => {
...
+): RouteWithQuote[][] => {
+  const thresholds = [60 * 1000, Infinity];
+  const buckets: RouteWithQuote[][] = [];
+
+  for (const routeAndQuote of routesWithQuotes) {
+    const { quote } = routeAndQuote;
+    for (let i = 0; i < thresholds.length; i++) {
+      const threshold = thresholds[i];
+      if (quote.eta && quote.eta <= threshold) {
+        if (buckets[i] === undefined) buckets[i] = [];
```
Comment (kev1n-peters) [inline wormhole-connect/src/hooks/useSortedRoutesWithQuotes.ts]: If we set `buckets[1]`, but never `buckets[0]`, then I think this call will throw when accessing `buckets[0]` which is `undefined`: https://github.com/wormhole-foundation/wormhole-connect/pull/2934/files#diff-0254f3d56b61ab2784afef85542f31c5480c00774bc2d46a93358cdd8077af6cR74

Comment (artursapek) [reply wormhole-connect/src/hooks/useSortedRoutesWithQuotes.ts]: oh good catch

Comment (artursapek) [reply wormhole-connect/src/hooks/useSortedRoutesWithQuotes.ts]: fixed in eef1044

**Review by kev1n-peters — APPROVED** [review]

**Comment by tsadovska** [comment]
Related issues:
* https://github.com/wormhole-foundation/wormhole-connect/issues/2949
* https://github.com/wormhole-foundation/wormhole-connect/issues/2954

**Pull request merged**